packages/nameguard-react/src/utils/text.ts

```diff
@@ -63,3 +64,7 @@ export function ratingTextColor(rating: Rating, isInteractive = false): string {
 
   return color;
 }
+
+export function getNameGuardURLForENSname(ensName: ENSName): string {
```

Comment:
```suggestion
export function viewNameReportURL(ensName: ENSName): URL {
```

1. Naming change suggested as this is the URL to view the NameGuard report for a name., not the "name guard url" or the "name guard url for ens name" as that can be confused as the API endpoint. Here we're specifically getting the URL to "viewNameReport". View being the key word here.
2. Seems nice to have this return a URL rather than a string.

I also don't think this utility function should be in `/utils/text.ts`. It should move to some other file under `/utils`. Open to ideas maybe `/utils/url.ts`